dashboard/app/main.go

```diff
@@ -533,7 +533,7 @@ func handleMain(c context.Context, w http.ResponseWriter, r *http.Request) error
 		BugFilter:      makeUIBugFilter(c, filter),
 	}
 
-	if isJSONRequested(r) {
+	if r.FormValue("json") != "" {
```

Comment:
Why is it better?

Reply:
Consistency.
1. We check form values in the code w/o helper functions.
2. To share this helper functions between syz-manager and dashboard/app, some shared pkg is needed. I don't see good candidates.